tests/misc_env/install_prereq.py

```diff
@@ -107,7 +110,9 @@ def install_prereq(
         # https://bugzilla.redhat.com/show_bug.cgi?id=1748015
         ceph.exec_command(cmd="sudo systemctl restart NetworkManager.service")
         if not skip_subscription:
-            setup_subscription_manager(ceph, is_production)
+            if cloud_type != "ibmc":
+                setup_subscription_manager(ceph, is_production)
+
             if not skip_enabling_rhel_rpms:
                 if enable_eus:
                     enable_rhel_eus_rpms(ceph, distro_ver)
```

Comment:
Are we missing here to pass cloud_type?

Reply:
Good catch. Thanks, I have amended the commit